hypothesis-python/tests/cover/test_statistical_events.py

```diff
@@ -260,3 +261,28 @@ def test(value):
 @given(st.booleans())
 def test_event_with_non_weakrefable_keys(b):
     event((b,))
+
+
+def test_assume_adds_event_with_function_origin():
+    @given(st.integers())
+    def very_distinguishable_name(n):
+        assume(n > 100)
+
+    stats = call_for_statistics(very_distinguishable_name)
+
+    for tc in stats["generate-phase"]["test-cases"]:
+        for event in tc["events"]:
```

Comment:
Linter is complaining that this loop variable (and below) is shadowing the import of `hypothesis.event`; rename?